app/assets/stylesheets/cards.css.scss

```diff
@@ -0,0 +1,61 @@
+
+$FLICKR_PINK: #ff2e92;
+$FLICKR_BLUE: #007bdc;
+
+$loader-size: 30px; 
+
+$loader-gutter: $loader-size/10;
+$loader-transform: $loader-size + $loader-gutter * 2;
+
+.loader {
+  position: relative;
+  width: $loader-size * 2;
+  height: $loader-size;
+  border-radius: $loader-size/2;
+  border: $loader-size * .75 solid transparent;
+  background: white;
+  margin: 0 auto 30px;
+}
+
+%loader-dot {
+  position: absolute;
```

Comment:
Properties should be ordered animation, border-radius, height, position, top, width